src/components/Personkort.tsx

```diff
@@ -57,7 +57,7 @@ export const PersonkortEPS = (props: { eps: Nullable<EPSMedAlder> }) => {
                 </span>
             </div>
             <div>
-                <p>{`${props.eps.navn.fornavn} ${props.eps.navn.mellomnavn} ${props.eps.navn.etternavn}`}</p>
+                <p>{`${props.eps.navn.fornavn} ${props.eps.navn.mellomnavn ?? ''} ${props.eps.navn.etternavn}`}</p>
```

Comment:
Her vil vi få et mellomrom for mye dersom man ikke har mellomnavn. Har vi ikke en hjelpefunksjon en plass for å formatere navn fint?